playground/express/index.js

```diff
@@ -24,12 +24,18 @@ Sentry.init({
   ]
 });
 
+setInterval(() => {
```

Comment:
Added for monitoring purposes. I ran a load test on a simple server for 100s with 50 concurrent connections to stress test and check for possible memory leaks.